testutil/retry/retry.go

```diff
@@ -0,0 +1,92 @@
+// Package retry provides support for repeating operations in tests.
+//
+// A sample retry operation looks like this:
+//
+//   func TestX(t *testing.T) {
+//       for r := retry.OneSec(); r.Next(t.FailNow); {
```

Comment:
I suppose you'd never implement the `Failer` interface just to log something - you'd just want to call some function.